test/unit/ManifestTest.php

```diff
@@ -31,61 +33,103 @@ class ManifestTest extends TestCase
     const MANIFEST_PATH_LIGHTWEIGHT = 'lightweightManifest.php';
     const MANIFEST_PATH_COMPLEX = 'complexManifest.php';
 
-    public function setUp(): void
+    private function getComposerInfoMock()
     {
+        $composerInfo = $this->getMockBuilder(ComposerInfo::class)->getMock();
+        $composerInfo->method('getComposerJson')->willReturn([
+            'name'=> 'oat-sa/extension-tao-lightweight',
+            'extra' => [
+                'tao-extension-name' => 'taoLightweight'
+            ],
+            'require'=> [
+                'oat-sa/oatbox-extension-installer'=> '~1.1||dev-master',
+                'oat-sa/extension-tao-taoItemBank'=> '*',
+                'oat-sa/extension-tao-taoDocuments'=> '*'
+            ]
+        ]);
+        $composerInfo->method('getPackageInfo')->will($this->returnValueMap(
+            [
+                [
+                    "oat-sa/extension-tao-lightweight", ROOT_PATH,
+                    [
+                        "name" => "oat-sa/extension-tao-lightweight",
+                        "version" => "v1.0"
+                    ]
+                ],
+                [
+                    "oat-sa/extension-tao-taoItemBank", ROOT_PATH,
+                    [
+                        "name" => "oat-sa/extension-tao-taoItemBank",
+                        "version" => "v1.0"
+                    ]
+                ],
+                [
+                    "oat-sa/extension-tao-taoDocuments", ROOT_PATH,
+                    [
+                        "name" => "oat-sa/extension-tao-taoDocuments",
+                        "version" => "v1.0"
+                    ]
+                ]
+            ]
+        ));
+
+        return $composerInfo;
     }
 
     public function testManifestLoading()
```

Comment:
We should probably split this test into multiple use cases "testManifestFailsLoadingIfFileNotFound", "testSimpleManifestLoadsCorrectly", "testComplexManifestLoadsCorrectly", ... to ease clarity

Reply:
Yes, make sense. But i just tried to make existing test work